### ISSUE TITLE
Camera module automatically shut down after starting, on the latest master branch

### ISSUE THREAD

**Issue by GabrielZZZ** [ApolloAuto/apollo#14598]
### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 18.04)**: Ubuntu 20.04
- **Apollo installed from (source or binary)**: source
- **Apollo version (3.5, 5.0, 5.5, 6.0)**: 7.0 master branch
- **Output of `apollo.sh config` if on `master` branch**:
`[INFO] Apollo Environment Settings:
[INFO]     APOLLO_ROOT_DIR: /apollo
[INFO]     APOLLO_CACHE_DIR: /apollo/.cache
[INFO]     APOLLO_IN_DOCKER: true
[INFO]     APOLLO_VERSION: master-2022-02-16-8fc0381
[INFO]     DOCKER_IMG: dev-x86_64-18.04-20210517_1712
[INFO]     APOLLO_ENV:  STAGE=dev USE_ESD_CAN=false
[INFO]     USE_GPU: USE_GPU_HOST=1 USE_GPU_TARGET=1
`
### Steps to reproduce the issue:
- **Please use bullet points and include as much details as possible**: 
- Successfully start, build Apollo without errors
- Strictly following the instructions from [https://github.com/ApolloAuto/apollo/blob/master/docs/howto/how_to_run_perception_module_on_your_local_computer.md](url)
- When running `cyber_launch start /apollo/modules/perception/production/launch/perception_camera.launch`, **the module first start, then automatically shut down. Here is the terminal output:**
![image](https://user-images.githubusercontent.com/42797233/190837913-b5acff63-4754-4921-86bb-1c80bd3bf57e.png)
Here is the log result from mainboard:
![image](https://user-images.githubusercontent.com/42797233/190837965-386ee980-5bcc-404d-9528-a2425be4e3e1.png)

I want to test the perception-camera module, especially the obstacle-detection capability.  I have a recording containing only images generated from Apollo DaoxiangLake Dataset.

The perception, traffic light module appears to be started successfully:
![image](https://user-images.githubusercontent.com/42797233/190838082-45696a1e-3711-4127-83d6-0aa03d4b65f9.png)

**Comment by ywangwxd** [comment]
I also encounered this problem. I solved it by launching the motion_service dag separately and remove the this module in this camera  perception *.launch file. 

I am not sure if duplicate node is the cause for this, because when I encountered this problem, I checked that if the motion_service not has been started or not.  And If a duplicate node is started, there will be very clear messgae telling you that the node is duplicate and it will not started. What is more, starting a duplicate node will not fail a cyber_launch, it will just tell you a duplicate node is found and it will be started again.

**Comment by ywangwxd** [comment]
I also encounered this problem. I solved it by launching the motion_service dag separately and remove the this module in this camera  perception *.launch file. 

I am not sure if duplicate node is the cause for this, because when I encountered this problem, I checked that if the motion_service not has been started or not.  And If a duplicate node is started, there will be very clear messgae telling you that the node is duplicate and it will not started. What is more, starting a duplicate node will not fail a cyber_launch, it will just tell you a duplicate node is found and it will be started again.
